libraries/cli/src/config.cpp

```diff
@@ -215,6 +230,22 @@ bool Config::nodeConfigured() { return node_configured_; }
 
 FullNodeConfig Config::getNodeConfiguration() { return node_config_; }
 
+void Config::addNewHardforks(Json::Value& config, const Json::Value& default_config) {
+  // network_id is exactly the same thing as chain_id. So get it from config
```

Comment:
is this comment still relevant? because I do not understand it

Reply:
Yeah, it should not be there